build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -29,5 +29,6 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="$(CLI_MSBuild_Version)" />
     <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(PlatformAbstractionsVersion)" />
     <PackageReference Include="Microsoft.DotNet.VersionTools" Version="$(VersionToolsVersion)" />
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="1.0.0-prerelease-01929-02" />
```

Comment:
This needs to be updated to latest.

Reply:
Can you elaborate on what you mean by "depends on buildtools" here?

Reply:
We are indeed on the latest version, there is just one version in the feed: https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/microsoft.dotnet.build.tasks.feed/1.0.0-prerelease-01929-02/